docs/running/slurm.md

```diff
@@ -38,6 +87,116 @@ This is the default partition, and will be used when you do not explicitly set a
 
 The following sections will provide detailed guidance on how to use SLURM to request and manage CPU cores, memory, and GPUs in jobs. These instructions will help users optimize their workload execution and ensure efficient use of CSCS computing resources.
 
+## Affinity
```

Comment:
The tool is great, but I think it might also be worth mentioning `srun`'s `--cpu-bind=verbose` option and the `OMP_DISPLAY_AFFINITY` environment variable in this section.

Reply:
Done